### PR TITLE
Fix js path of oneof field

### PR DESCRIPTION
**Pull request by egormodin**
Due to our limited support resources, we are focusing our attention on small bug fixes that have limited compatibility concerns. Most other PRs will be rejected.

### CODE REVIEW

File: generator/js_generator.cc [hunk 1]
```diff
@@ -2254,7 +2254,7 @@ void Generator::GenerateClassXid(const GeneratorOptions& options,
 
 void Generator::GenerateOneofCaseDefinition(
     const GeneratorOptions& options, io::Printer* printer,
-    const OneofDescriptor* oneof) const {
+    const OneofDescriptor* oneof, const Descriptor* desc) const {
```
Comment (reddaly) [inline generator/js_generator.cc]: What is needed is probably changing className = GetMessagePath to `oneof->containing_type()->name())`. I don't think taking `desc` is necessary.

File: generator/js_generator.cc [hunk 2]
```diff
@@ -2237,7 +2237,7 @@ void Generator::GenerateClassFieldInfo(const GeneratorOptions& options,
       if (IgnoreOneof(desc->oneof_decl(i))) {
         continue;
       }
-      GenerateOneofCaseDefinition(options, printer, desc->oneof_decl(i));
+      GenerateOneofCaseDefinition(options, printer, desc->oneof_decl(i), desc);
```
Comment (reddaly) [inline generator/js_generator.cc]: There should be no need to pass desc since it can be obtained from the OneofDescriptor.

Comment (egormodin) [reply generator/js_generator.cc]: done

**Comment by gonzojive** [comment]
This change seems to have been applied already.

**Pull request closed without merging**